src/e2e/__tests__/caret.ts

```diff
@@ -83,8 +91,11 @@ describe('caret testing', () => {
 
     await page.keyboard.press('Enter')
     await paste(page, [''], importText)
-    await setCursor(page, ['Purple Rain'], { offset: 5 })
-    const editableNodeHandle = await getEditable(page, 'Purple Rain')
+
+    const editableNodeHandle = await waitForEditable(page, 'Purple Rain')
+
+    await clickWithOffset(page, editableNodeHandle, { horizontalClickLine: 'left', offset: 5 })
```

Comment:
`offset` overrides `horizontalClickLine`. So you can remove `horizontalClickLine` here.